### PR TITLE
feat: Support Json variable in binding

### PR DESCRIPTION
**Pull request by benjaminParisel**
* We can use a JSON variable in the model and bind it as interpolation or variable
* Update value with a set is reflected in model

Covers[UID-553](https://bonitasoft.atlassian.net/browse/UID-553)

### CODE REVIEW

**Review by jeromecambon — CHANGES_REQUESTED** [review]
Seems good overall !
First set of comments (before testing)

File: src/resolvers/JsonResolver.ts [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+    watchDependencies(){}
+
+    resolve() : void {
+        try{
+            Object.defineProperty(this.model,this.name,  {
+                value: JSON.parse(this.content),
+                writable: true
+            });
+        } catch(e){
```
Comment (jeromecambon) [inline src/resolvers/JsonResolver.ts:18]: Seems a JSON error (e.g. bad syntax) is not catched here, and we never get the error message in the console

**Review by jeromecambon — COMMENTED** [review]
I tested with different json objects with quite complex structure: this works fine :-)
See just the comment on error handling in JsonResolver.

File: src/resolvers/JsonResolver.ts (hunk 1 above)
Comment (benjaminParisel) [reply src/resolvers/JsonResolver.ts:16]: A test in JsonResolver.spec test if catch is worked

File: src/bindingType/Binding.ts [hunk 2]
```diff
@@ -1,8 +1,12 @@
-import { Property, VariableContext } from '../ContextBindingType';
-import { VariableAccessor } from '../VariableAccessor';
+import {Property, VariableContext} from '../ContextBindingType';
+import {VariableAccessor} from '../VariableAccessor';
 import evaluate from 'ts-expression-evaluator';
```
[the remark this answers is not in the document]
Comment (benjaminParisel) [reply src/bindingType/Binding.ts:3]: Yes for evalutation js expression

**Pull request merged**